dev/tools/svg2dart/bin/main.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:args/args.dart';
+
+void main(List<String> args) {
+  ArgParser parser = new ArgParser();
+
+  parser.addFlag(
+      'help',
+      abbr: 'h',
+      negatable: false,
+      help: 'Display the tool\'s usage instructions and quit');
+
+  parser.addOption(
+      'output',
+      abbr: 'o',
+      help: 'Target path to write the generated Dart file to');
+
+  parser.addOption(
+      'asset_name',
+      abbr: 'n',
+      help: 'Name to be used ');
```

Comment:
trivial nit: extraneous space

also, this function isn't wrapped to flutter style :-)